src/content/docs/en/guides/cms/frontmatter-cms.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: Front Matter CMS & Astro
+description: Add content to your Astro project using Front Matter CMS
+type: cms
+service: Front Matter CMS
+i18nReady: false
+---
+import FileTree from '~/components/FileTree.astro'
+
+[Front Matter CMS](https://frontmatter.codes/) brings the CMS to your editor, it runs within Visual Studio Code, GitPod, and many more.
+
+## Integration with Astro
+
+In this section, we'll walk through how to add Front Matter CMS to your Astro project.
+
+### Prerequisites 
+
+- Visual Studio Code
+- Project with content, a great start is the [Astro Blog template](https://github.com/withastro/astro/tree/main/examples/blog)
+
+### Install the Front Matter CMS extension
+
+You can get the extension from the [Visual Studio Code Marketplace - Front Matter](https://marketplace.visualstudio.com/items?itemName=eliostruyf.vscode-front-matter) or by clicking on the following link: [open Front Matter CMS extension in VS Code](vscode:extension/eliostruyf.vscode-front-matter)
+
+### Project initialization
+
+Once Front Matter CMS is installed, you will get a new icon in the Activity Bar on the left side (or right side depending on your preference) of the editor. Click on it to open the **Front Matter CMS** panel.
+
+- Click on the **Initialize project** button;
+- The welcome screen will open and you can start initializing the project;
+- Click on the first step to **Initialize project**;
+- As Astro is one of the supported frameworks, you can select it from the list;
+- Register your content folders, in this case, the `src/content/blog` folder;
+- Click on **Show the dashboard** to open the content dashboard.
```

Comment:
Remove the semicolons - the bullets are enough!

Reply:
I learned it in school as a best practice 🤣. I'll remove them.